api/swagger.yaml

```diff
@@ -189,6 +189,48 @@ definitions:
       - CICD
       - RUNTIME
 
+  CISDockerBenchmarkLevelCount:
```

Comment:
maybe CISDockerBenchmarkIssuesPerLevelCount
and to add description:
Number of CIS Docker benchmark issues for a specific level